pkg/policy/l4.go

```diff
@@ -130,16 +130,17 @@ func (l4 *L4Filter) HasL3DependentL7Rules() bool {
 		// No L7 rules.
 		return false
 	case 1:
-		// If L3 is wildcarded, this filter corresponds to L4-only rule(s).
-		_, ok := l4.L7RulesPerEp[api.WildcardEndpointSelector]
-		return !ok
-	default:
-		return true
+		// loop to get access to the first and only key in the map
```

Comment:
To confirm - this is relying on the semantics that if the wildcard selector is in `L7RulesPerEp`, it will be the only key in `L7RulesPerEp` ?